fdbclient/FDBAWSCredentialsProviderChain.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * FDBAWSCredentialsProviderChain.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef BUILD_AWS_BACKUP
+
```

Comment:
Add something like `#ifndef xxx` to make sure this header file is only included once.

Reply:
thanks, will update the PR once I discuss testing with @johscheuer